pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -19,139 +19,137 @@ import (
 	"context"
 	"encoding/base64"
 	"fmt"
-	"strings"
 	"text/template"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
-	"github.com/aws/aws-sdk-go/service/ssm"
-	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
 	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
 	"github.com/mitchellh/hashstructure/v2"
 
 	"github.com/patrickmn/go-cache"
 	"go.uber.org/zap"
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/client-go/kubernetes"
 )
 
 const (
-	launchTemplateNameFormat = "Karpenter-%s/%s/%s-%s"
+	launchTemplateNameFormat = "Karpenter-%s-%s"
```

Comment:
Is there a general launch template name format that we should follow here? Are forwards slashes not how they're normally created, or is it mainly hyphenated?

Reply:
I haven't seen a consistent format or pattern for this. Luckily it's easy to change in a backwards compatible way.